corehq/apps/export/templates/export/partials/table.html

```diff
@@ -362,23 +362,41 @@ <h4 data-bind="text: case_type"></h4>
           <div data-bind="ifnot: can_edit()">
             <div data-bind="if: isLocationSafeForUser">
               <a class="btn btn-default disabled" data-bind="visible: $parent.myExports">
-                <i class="fa fa-pencil"></i>
-                <span data-bind="visible: !isDailySaved()">{% trans 'Edit' %}</span>
-                <span data-bind="visible: isDailySaved()">{% trans 'Edit Columns' %}</span>
+                <div data-bind="ifnot: isOData()">
+                  <i class="fa fa-pencil"></i>
+                  <span data-bind="visible: !isDailySaved()">{% trans 'Edit' %}</span>
+                  <span data-bind="visible: isDailySaved()">{% trans 'Edit Columns' %}</span>
+                </div>
+                <div data-bind="if: isOData()">
+                  <i class="fa fa-pencil"></i>
+                  <span>{% trans 'Copy Feed' %}</span>
+                </div>
               </a>
               <a data-bind="attr: {href: editUrl}, visible: !$parent.myExports" class="btn btn-default">
-                <span data-bind="visible: !isDailySaved()">{% trans 'View' %}</span>
-                <span data-bind="visible: isDailySaved()">{% trans 'View Columns' %}</span>
+                <div data-bind="ifnot: isOData()">
+                  <span data-bind="visible: !isDailySaved()">{% trans 'View' %}</span>
+                  <span data-bind="visible: isDailySaved()">{% trans 'View Columns' %}</span>
+                </div>
+                <div data-bind="if: isOData()">
+                  <i class="fa fa-pencil"></i>
+                  <span>{% trans 'View Feed' %}</span>
+                </div>
               </a>
             </div>
           </div>
         </div>
         <div data-bind="ifnot: $parent.showOwnership">
           <div data-bind="if: isLocationSafeForUser">
             <a data-bind="attr: {href: editUrl}" class="btn btn-default">
-              <i class="fa fa-pencil"></i>
-              <span data-bind="visible: !isDailySaved()">{% trans 'Edit' %}</span>
-              <span data-bind="visible: isDailySaved()">{% trans 'Edit Columns' %}</span>
+              <div data-bind="ifnot: isOData()">
+                <i class="fa fa-pencil"></i>
+                <span data-bind="visible: !isDailySaved()">{% trans 'Edit' %}</span>
+                <span data-bind="visible: isDailySaved()">{% trans 'Edit Columns' %}</span>
+              </div>
+              <div data-bind="if: isOData()">
+                <i class="fa fa-pencil"></i>
+                <span>{% trans 'Copy Feed' %}</span>
```

Comment:
Another approach to this would be to add `edit_button_label` as a parameter to the view context, similar to `shared_export_type` and `export_type_caps`